### DISCUSSION TITLE
Porque utilizar PO UI no Fluig?

### DISCUSSION THREAD

**Discussion by developerWelligton** [po-ui/po-angular#1699] [Assuntos gerais]
Porque utilizar PO UI no Fluig?

**Comment by jkozima** [comment]
Boa Tarde @developerWelligton 
Tudo bem?
A utilização do PO UI no Fluig pode contribuir para a criação de aplicações mais eficientes, intuitivas e consistentes visualmente, além de reduzir o tempo e o esforço necessários para o desenvolvimento de interfaces de usuário.
